src/Flicking.ts

```diff
@@ -668,7 +668,7 @@ class Flicking extends Component {
       if (options.circular && looped) {
         // Reached at max/min range of axes
         const scrollAreaSize = viewport.getScrollAreaSize();
-        cameraChange = -Math.sign(cameraChange) * (scrollAreaSize - Math.abs(cameraChange));
+        cameraChange = -(cameraChange > 0 ? 1 : -1) * (scrollAreaSize - Math.abs(cameraChange));
```

Comment:
@daybrush It's not import issue.but I suggest `cameraChange < 0 ? -1 : 1` because 0 could be treated as negative.